include/xtensor-zarr/xzarr_attrs.hpp

```diff
@@ -0,0 +1,40 @@
+/***************************************************************************
+* Copyright (c) Wolf Vollprecht, Sylvain Corlay and Johan Mabille          *
+* Copyright (c) QuantStack                                                 *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#ifndef XTENSOR_ZARR_XZARR_ATTRS_HPP
+#define XTENSOR_ZARR_XZARR_ATTRS_HPP
+
+#include "nlohmann/json.hpp"
+#include "xtensor/xchunked_array.hpp"
+
+namespace xt
+{
+    class xzarr_attrs
```

Comment:
I think this class could live in `xzarr_hierarchy` (except if you plan to use it as a base class of other expressions later). Also it could be nice to ad a short comment explaning that this class is expected to be used as a base class extension of xtensor expressions.